Synchronize WPT interfaces partially

annevk opened the pull request:
#### 443e0896e6d51e7e755786265e6ff447a63a595a
<pre>
Synchronize WPT interfaces partially
<a href="https://bugs.webkit.org/show_bug.cgi?id=271639">https://bugs.webkit.org/show_bug.cgi?id=271639</a>

Reviewed by Darin Adler.

Updates the IDL of various standards up to this commit:
<a href="https://github.com/web-platform-tests/wpt/commit/cea69e33dac7011aeda11cb114ce58e283e0ffe2">https://github.com/web-platform-tests/wpt/commit/cea69e33dac7011aeda11cb114ce58e283e0ffe2</a>

* LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/idlharness.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/encoding/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/encoding/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fs/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/OWNERS: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/ResizeObserver.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/WebCryptoAPI.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/console.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/dom.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/encoding.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/fetch.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/fs.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/hr-time.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/notifications.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/origin-policy.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/pointerevents-extension.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/remoteplayback.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/reporting.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/requestidlecallback.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/resize-observer.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/service-workers.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/storage-buckets.tentative.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/streams.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/ua-client-hints.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/uievents.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/url.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/wai-aria.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/xhr.idl:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/uievents/idlharness.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/url/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/url/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/wai-aria/idlharness.window-expected.txt:

Canonical link: <a href="https://commits.webkit.org/276658@main">https://commits.webkit.org/276658@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/654e3ad22f35639fe3844c13554a06b8c39fd48b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/45245 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/24363 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/47765 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/47943 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/41253 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/28552 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/21760 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/37131 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/45823 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/21417 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/39022 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/18231 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/18830 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/40105 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/3292 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/41513 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/40422 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/49613 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/20227 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/16757 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/44142 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/21533 "Built successfully") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/42943 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/10066 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/21894 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/21221 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

darinadler reviewed (approved):
So great to do this. I’m always tempted to fix any new `FAIL` when I do a task like this, but I can’t in good conscience suggest that you do so at this time.

annevk committed:
Synchronize WPT interfaces partially
https://bugs.webkit.org/show_bug.cgi?id=271639

Reviewed by Darin Adler.

Updates the IDL of various standards up to this commit:
https://github.com/web-platform-tests/wpt/commit/cea69e33dac7011aeda11cb114ce58e283e0ffe2

* LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/idlharness.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/idlharness.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/encoding/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/encoding/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fetch/api/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/fs/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/OWNERS: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/ResizeObserver.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/WebCryptoAPI.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/console.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/dom.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/encoding.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/fetch.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/fs.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/hr-time.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/notifications.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/origin-policy.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/pointerevents-extension.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/remoteplayback.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/reporting.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/requestidlecallback.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/resize-observer.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/service-workers.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/storage-buckets.tentative.idl: Removed.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/streams.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/ua-client-hints.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/uievents.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/url.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/wai-aria.idl:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/xhr.idl:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/service-workers/idlharness.https.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.serviceworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.sharedworker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/streams/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/uievents/idlharness.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/url/idlharness.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/url/idlharness.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/wai-aria/idlharness.window-expected.txt:

Canonical link: https://commits.webkit.org/276658@main
